packages/api/src/webviewWindow.ts

```diff
@@ -75,21 +79,21 @@ class WebviewWindow {
   constructor(
     label: WebviewLabel,
     options: Omit<WebviewOptions, 'x' | 'y' | 'width' | 'height'>
-      & WindowOptions = {}
+      & WindowOptions & { skip?: boolean } = {}
```

Comment:
This will be visible in the public API, which is why we didn't do it in the first place, as this concerns internal usage only.